### PR TITLE
Extend Metrics test about testing uptime and start date

### PR DESCRIPTION
**Pull request by mkralik3**
* Test for Start Date was added
 * Test for Uptime was added

//test: `@integration-metrics`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/other/integration-metrics.feature [hunk 1]
```diff
@@ -234,4 +243,6 @@ Feature: Metrics
     And check that last processed date is valid
     And check that number of valid messages is 1
     And check that number of error messages is 0
-    And check that number of total messages is 1
\ No newline at end of file
+    And check that number of total messages is 1
+    And check that uptime for webhook-to-db pod is valid
+    And check that startdate for webhook-to-db pod is valid
```
Comment (avano) [inline ui-tests/src/test/resources/features/other/integration-metrics.feature]: please add a new line here, otherwise it's ok

Comment (mkralik3) [reply ui-tests/src/test/resources/features/other/integration-metrics.feature]: added

**Comment by avano** [comment]
@mkralik3 probably the IDE added 4 spaces instead of an empty new line :-)

**Comment by avano** [comment]
seems like the time on the machines are synced:

```
[root@master ~]# date
Út led 15 02:31:19 EST 2019
[root@master ~]# date
Út led 15 02:31:21 EST 2019
[root@master ~]# date
Út led 15 02:31:22 EST 2019
[root@master ~]# date
Út led 15 02:31:23 EST 2019
[root@master ~]# date
Út led 15 02:31:24 EST 2019
[root@master ~]# date
Út led 15 02:31:24 EST 2019
[root@master ~]# date
Út led 15 02:31:25 EST 2019
```

```
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:19 EST 2019
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:21 EST 2019
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:22 EST 2019
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:23 EST 2019
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:24 EST 2019
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:24 EST 2019
[hudson@rhel7-ui-11004 ~]$ date
Út led 15 02:31:25 EST 2019
```

**Comment by mkralik3** [comment]
UI doesn't contain seconds, so maybe it takes the time from the OpenShift and round-up minutes or something. I have to look at the implementation.

**Comment by mkralik3** [comment]
I found it. My bad. Method [DateUtils.round()](https://github.com/syndesisio/syndesis-qe/pull/677/commits/cdafee342dc697ba1c55e232d169ef336a87980b#diff-1feedfa36df32128ecd1dadaf2661e7eR166) returns a rounded Date as a return value and doesn't modify the original as I wrote. 
~~But I don't understand that it worked until now.~~
OK, I got it. It is a UI issue. When you will push F5, sometimes the "since" time is changed to actual time and vice versa. I am working on workaround in the test.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mkralik3** [comment]
Rebased with master and resolved conflict.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**